src/app/modules/form/admin-form/admin-form.utils.ts

```diff
@@ -586,3 +588,36 @@ export const mapGoGovErrors = (error: AxiosError): GoGovError => {
       )
   }
 }
+
+// TODO: remove once we upgrade to Mongoose 7.x
+/**
+ * Manually insert default validation options for short text columns in tables.
+ * Due to a bug in Mongoose 6.x, default values are not inherited for discriminators.
+ *
+ * See https://github.com/Automattic/mongoose/issues/12135
+ *
+ * Fixed in Mongoose 7.x
+ * @param newField
+ * @returns
+ */
+
+export const insertTableShortTextColumnDefaultValidationOptions = (
+  newField: FieldUpdateDto,
+) => {
+  if (newField.fieldType === BasicField.Table) {
+    const defaultValidationOptions = {
+      customVal: null,
+      selectedValidation: null,
```

Comment:
I'm wondering if we should add the prefill properties too, just for db consistency's sake

Reply:
Hm those are marked as [optional](https://github.com/opengovsg/FormSG/blob/develop/shared/types/field/shortTextField.ts) on types. But if we look at inheritance it should be populated. 

Both are correct. I in favour of keeping it as simple as possible.